src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import core.basesyntax.fileoperations.FileReader;
+import core.basesyntax.fileoperations.FileReaderImpl;
+import core.basesyntax.fileoperations.FileWriter;
+import core.basesyntax.fileoperations.FileWriterImpl;
+import core.basesyntax.fileoperations.ReportGenerator;
+import core.basesyntax.fileoperations.ReportGeneratorImpl;
+import core.basesyntax.operations.BalanceOperation;
+import core.basesyntax.operations.OperationHandler;
+import core.basesyntax.operations.OperationStrategy;
+import core.basesyntax.operations.OperationStrategyImpl;
+import core.basesyntax.operations.PurchaseOperation;
+import core.basesyntax.operations.ReturnOperation;
+import core.basesyntax.operations.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] arg) {
```

Comment:
The `main` method should not begin with an empty line according to the checklist.